package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apigeetool",
-  "version": "0.11.0",
+  "version": "0.12.0",
```

Comment:
Please don't bump this, we'll bump it as part of our release process.

Reply:
@whitlockjc I've reverted this change